src/log-analytics/azext_loganalytics/tests/latest/test_loganalytics_commands.py

```diff
@@ -27,7 +27,7 @@ def test_query(self, resource_group):
             'az monitor log-analytics query -w {workspace_customerId} '
             '--analytics-query "Heartbeat | getschema"',
             checks=[
-                self.check("length(@)", 30),
+                self.check("length(@)", 31),
```

Comment:
the return schema changed?

Reply:
yes, there's a new property `ComputerPrivateIPs`